dev/ScrollPresenter/APITests/ScrollPresenterTests.cs

```diff
@@ -879,6 +879,40 @@ public void ListenToContentEffectiveViewportChanged()
             });
         }
 
+        [TestMethod]
+        [TestProperty("Description", "ScrollPresenter should handle BringIntoView for its direct content.")]
+        public void HandleContentBringIntoView()
```

Comment:
This test (I renamed and fixed it) should be added to ScrollPresenterBringIntoViewTests.cs where bring-into-view tests are all located:

```
        [TestMethod]
        [TestProperty("Description", "ScrollPresenter must handle BringIntoView request for its direct Content with a large Margin.")]
        public void BringContentWithMarginIntoView()
        {
            ScrollPresenter scrollPresenter = null;
            Shapes.Rectangle rectangleScrollPresenterContent = null;
            AutoResetEvent scrollPresenterViewChangedEvent = new AutoResetEvent(false);
            AutoResetEvent scrollPresenterLoadedEvent = new AutoResetEvent(false);

            RunOnUIThread.Execute(() =>
            {
                rectangleScrollPresenterContent = new Shapes.Rectangle()
                {
                    Margin = new Thickness(0, 500, 0, 0)
                };
                scrollPresenter = new ScrollPresenter();

                SetupDefaultUI(scrollPresenter, rectangleScrollPresenterContent, scrollPresenterLoadedEvent);
            });

            WaitForEvent("Waiting for Loaded event", scrollPresenterLoadedEvent);

            RunOnUIThread.Execute(() =>
            {
                scrollPresenter.ViewChanged += delegate (ScrollPresenter sender, object args)
                {
                    Log.Comment("ViewChanged - HorizontalOffset={0}, VerticalOffset={1}, ZoomFactor={2}",
                        sender.HorizontalOffset, sender.VerticalOffset, sender.ZoomFactor);
                    scrollPresenterViewChangedEvent.Set();
                };

                Log.Comment("Calling ScrollPresenter.Content.StartBringIntoView");
                rectangleScrollPresenterContent.StartBringIntoView(new BringIntoViewOptions() { AnimationDesired = false });
            });

            WaitForEvent("Waiting for ViewChanged event", scrollPresenterViewChangedEvent);

            RunOnUIThread.Execute(() =>
            {
                Verify.IsGreaterThan(scrollPresenter.VerticalOffset, 0.0);
            });
        }

```

Reply:
Ah, didn't notice this and fixed it also 😀  . Anyway, at least fixed local debugging of tests. I will move the tests to the correct location, thanks!